src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/actionfurtherevidence/ActionFurtherEvidenceReceivedHandler.java

```diff
@@ -0,0 +1,50 @@
+package uk.gov.hmcts.reform.sscs.ccd.presubmit.actionfurtherevidence;
+
+import static java.util.Objects.requireNonNull;
+import static uk.gov.hmcts.reform.sscs.ccd.presubmit.actionfurtherevidence.FurtherEvidenceActionDynamicListItems.*;
+
+import org.apache.commons.lang.StringUtils;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.sscs.ccd.callback.Callback;
+import uk.gov.hmcts.reform.sscs.ccd.callback.CallbackType;
+import uk.gov.hmcts.reform.sscs.ccd.callback.PreSubmitCallbackResponse;
+import uk.gov.hmcts.reform.sscs.ccd.domain.*;
+import uk.gov.hmcts.reform.sscs.ccd.presubmit.PreSubmitCallbackHandler;
+
+
+@Service
+public class ActionFurtherEvidenceReceivedHandler implements PreSubmitCallbackHandler<SscsCaseData> {
```

Comment:
I find all these class names relating to the action further evidence event quite confusing now. Could we agree a pattern and try and use this? Happy to discuss...